Update data.json

san-npm opened the pull request:
Aleph Cloud added to the repo

Jimmy-Ma-BNBChain reviewed (changes requested):
The "tags" value is invalid

san-npm committed:
Add Aleph Cloud to developer tools list

Decentralized cloud (storage, VPS, compute, TEE, AI), EVM + SVM
compatible, BSC support. tags="ALL,AI" (both valid values).
